includes/object-cache.php

```diff
@@ -2417,7 +2417,21 @@ public function add_global_groups( $groups ) {
      * @param array $groups  List of groups that are to be ignored.
      */
     public function add_non_persistent_groups( $groups ) {
-        $groups = (array) $groups;
+        $groups          = (array) $groups;
+        $default_groups  = [ 'counts', 'plugins', 'themes' ];
```

Comment:
Not stoked about keeping track here of these. I rather ditch `$this->ignored_groups` and get everything from core.

Reply:
I agree it's not ideal but it's the only way I can think of handling this that doesn't break backwards compatibility.

Reply:
Okay, let's at least use the class property.

Reply:
Done :heavy_check_mark: